src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -187,6 +188,50 @@ public void Clear()
             }
         }
 
+        public TValue GetOrAdd(TKey key, TValue value)
```

Comment:
Please add XML comments for all of the public surface area.

Reply:
Yup yup on it, accidentally opened not as a draft 😅

Reply:
Updated!